docs/_data/cards.yml

```diff
@@ -121,32 +122,34 @@
   passives:
     - name: Pure Angel
       levels:
-        - When using the blue orb, restores <b>1</b> additional MP.
-          All allies gain a barrier absorbing <b>10%</b> of maximum HP for 6 turns.
-        - When using the blue orb, restores <b>2</b> additional MP.
-          All allies gain a barrier absorbing <b>15%</b> of maximum HP for 6 turns.
-        - When using the blue orb, restores <b>3</b> additional MP.
-          All allies gain a barrier absorbing <b>20%</b> of maximum HP for 6 turns.
-    - name: 天使の爛漫
+        - When using the blue orb, generates <b>1</b> additional MP.
+          All allies gain a barrier that absorbs up to <b>10%</b> of maximum HP for 6 turns.
+        - When using the blue orb, generates <b>2</b> additional MP.
+          All allies gain a barrier that absorbs up to <b>15%</b> of maximum HP for 6 turns.
+        - When using the blue orb, generates <b>3</b> additional MP.
+          All allies gain a barrier that absorbs up to <b>20%</b> of maximum HP for 6 turns.
+    - name: Angel in Bloom
       levels:
         - When any ally receives a critical hit, they restore <b>10%</b> of the damage taken, then gain <b>1%</b> critical damage reduction (stacking up to 10 times).
         - When any ally receives a critical hit, they restore <b>12.5%</b> of the damage taken, then gain <b>2%</b> critical damage reduction (stacking up to 10 times).
         - When any ally receives a critical hit, they restore <b>15%</b> of the damage taken, then gain <b>3%</b> critical damage reduction (stacking up to 10 times).
-
+  notes:
+    Strongest Ally: Witch with the highest offensive stat total (Physical + Magic).
+    Weakest Ally: Witch with the lowest current HP stat.
 ##
 ## Sakura
 ##
 - witch: Millefeuille
   release: Sakura
   rarity: UR
-  _title: 花風に揺られて
+  _title: Fluttering Petals in the Wind
   tags:
     - AoE Attack
     - Debuff
   hidden_tags:
     - Poison
   spell:
-    name: 桜花流光
+    name: Light Scattering Cherry Blossoms
```

Comment:
Probably should be **Light-scattering** (i.e., blossoms that scatter light, not light that scatters blossoms)?